models/olids/conformed/conformed_appointment.sql

```
@@ -54,32 +54,20 @@ SELECT
-    src.lds_lakehouse_date_processed,
-    src.lds_lakehouse_datetime_updated
-FROM {{ source('olids_common', 'APPOINTMENT') }} src
+    src.publisher_organisation_code,
+    src.source_extraction_date,
+    src.lds_transform_datetime
+FROM {{ ref('landing_appointment') }} src
 INNER JOIN {{ ref('base_olids_patient') }} patients
     ON src.patient_id = patients.id
 INNER JOIN {{ ref('int_wnl_practices') }} wnl_practices
     ON src.publisher_organisation_code = wnl_practices.practice_code
 LEFT JOIN {{ ref('int_enriched_concept_map') }} appointment_status_map
```

SQLFluff (4.2.2)

[error] Implicit/explicit aliasing of table.

(AL01)